CMakeLists.txt

```diff
@@ -16,7 +16,10 @@ find_package(ignition-cmake2 2.13.0 REQUIRED)
 #============================================================================
 set (c++standard 17)
 set (CMAKE_CXX_STANDARD 17)
-ign_configure_project(VERSION_SUFFIX)
+ign_configure_project(
+  REPLACE_IGNITION_INCLUDE_PATH gz/math
```

Comment:
I'm not completely sure about the consequences of this change, could this break anyone downstream, @scpeters ?

Reply:
this changes where the configured header files (`math.hh`, `math/config.hh`, `math/Export.hh`, etc) are installed. The redirection headers installed in `include/ignition` should keep working any existing code that includes `ignition/math*`. I think it's safe to do; the abichecker job is happy